packages/earth-map/src/components/collection/collection-editplaces/CollectionEditPlaces.tsx

```diff
@@ -96,35 +100,64 @@ export function CollectionEditPlaces(props: IProps) {
             {t('Cancel')}
           </button>
         </Card>
+
+        {isSaveConflict && <CollectionConflict onRefresh={refresh} onOverwrite={saveAnyway} />}
       </div>
     </form>
   );
 
-  async function onSubmit(values) {
+  /**
+   * Receives event when called by react-hook-form and boolean when called by us
+   */
+  async function onSubmit(values, optional: BaseSyntheticEvent | boolean) {
+    const shouldOverwrite = isBoolean(optional);
     const parsedValues = {
       ...values,
 
       // The api expects an array of ids or an empty array
       // should this be handled by AsyncSelect?
       ...(values.locations ? { locations: values.locations.map((x) => x.id) } : { locations: [] }),
+      // Sending the version to the backend will kick in the version validation
+      // To keep the api backwards compatible, when no version is passed, we overwrite
+      ...(!shouldOverwrite && { version }),
     };
 
     try {
-      const { data } = await PlacesService.updateCollection(id, parsedValues, {
+      const { data } = await PlacesService.updateCollection(slug, parsedValues, {
```

Comment:
We had a few conversations, should we still use the `id` here?

If somebody changes the collection, and updates the slug, I should still be able to get the prompt to:
- update to latest + redirect to the new slug;
- or, overwrite collection with my data (and slug);

Reply:
I think whichever option works, maybe it's better to not allow the users to change the slug so much.